client/src/components/routes/events/NewEvent.js

```diff
@@ -191,6 +191,7 @@ class NewEvent extends Component {
           <UploadArea>
             <label>
               <AppTextP>Upload an Image</AppTextP>
+              <AppTextP size="0.05">NOTE: The image must be an 11 x 11 inch .png file.</AppTextP>
```

Comment:
My mistake, it should be 11 inches by 17 inches. Please change accordingly

Reply:
Also, to make it clearer, it should be "11 inches by 17 inches" instead of just "11 x 11 inch"``

Reply:
Edited!